crates/solidity/outputs/cargo/crate/src/backend/ir/mod.rs

```diff
@@ -1,2 +1,4 @@
 pub mod ir1_structured_ast;
 pub mod ir2_flat_contracts;
+
+pub mod ast;
```

Comment:
WDYT of renaming `ast` to `ir3_public_ast` to indicate its order/place in the pipeline?

Reply:
I'm not sure about this. The AST shouldn't be another intermediate representation, it's a terminal representation. We may have an IR3 derived from IR2 in the future if it makes sense to accommodate the requirements of the backend later. I see the AST as a spin off from the IRs. In fact, I was thinking about move the module either into the `semantic` or to the parent `backend` module.